authority/provisioner/acme.go

```diff
@@ -392,3 +442,12 @@ func (p *ACME) IsAttestationFormatEnabled(_ context.Context, format ACMEAttestat
 func (p *ACME) GetAttestationRoots() (*x509.CertPool, bool) {
 	return p.attestationRootPool, p.attestationRootPool != nil
 }
+
+// IsRootRevoked return a true if the serialNumber is part of the list
+// It will also be in charge of updating the list periodically if no CRL list is provided at configuration.
+func (p *ACME) IsRootRevoked(serialNumber string) bool {
```

Comment:
Also see comment w.r.t. fetching the serials. It would be better to extract that out of this function, to make it responsible for a single thing.
```suggestion
// IsRootRevoked returns true if the provided serialNumber is in the list of revoked 
// certificate serial number.
// It will also be in charge of updating the list periodically if no CRL list is provided at configuration.
func (p *ACME) IsRootRevoked(serialNumber string) bool {
```